index.html

```diff
@@ -13219,16 +13219,20 @@ <h2>Excluding Elements from the Accessibility Tree</h2>
+          <p>If not excluded from the accessibility tree per the rules above in <a href="#tree_exclusion">Excluding Elements in the Accessibility Tree</a>, user agents MUST provide an <a>accessible object</a> in the <a>accessibility tree</a> for <abbr title="Document Object Model">DOM</abbr> <a>elements</a> that meet any of the following criteria:</p>
           <ul>
-            <li>Text nodes</li>
-            <li>Elements that may fire an <a>accessibility <abbr title="Application Program Interface">API</abbr></a> <a>event</a>.</li>
-            <li>Elements that are focusable, or have an ID <a>attribute</a> and an ancestor with the <pref>aria-activedescendant</pref> attribute that matches the implicit or explicit <a>semantics</a> of the required context role. In either case, the element may receive focus and may fire an <a>accessibility <abbr title="Application Program Interface">API</abbr></a> focus event.</li>
+            <li>Elements that are not <a>hidden</a> and may fire an <a>accessibility <abbr title="Application Program Interface">API</abbr></a> <a>event</a>:
+              <ul>
+                <li>Elements that are focusable even if the element or one of its ancestor elements has its <sref>aria-hidden</sref> attribute set to <code>true</code>.</li>
+                <li>Elements that have an ID <a>attribute</a>, and are contained in or <a>owned</a> by an element that supports and has the <pref>aria-activedescendant</pref> property, and have an implicit or explicit role that matches the required context role of their containing element, and their containing element is either focusable or is controlled by an element that is focusable.</li>
+              </ul>
+	        </li>
             <li>Elements that have an explicit role or a global <abbr title="Accessible Rich Internet Application">WAI-ARIA</abbr> attribute and do not have <sref>aria-hidden</sref> set to <code>true</code>. (See <a href="#tree_exclusion">Excluding Elements in the Accessibility Tree</a> for additional guidance on <sref>aria-hidden</sref>.)</li>
             <li>Elements that have an ID which is referenced by another element via a <abbr title="Accessible Rich Internet Application">WAI-ARIA</abbr> property and are not <a>hidden</a>.
             	<p class="note">Text equivalents for hidden referenced objects may still be used in the <a href="https://www.w3.org/TR/accname-aam-1.1/#mapping_additional_nd">name and description calculation</a> [[ACCNAME-1.1]] even when not included in the accessibility tree.</p>
             </li>
           </ul>
+          <p>User agents MAY provide an <a>accessible object</a> in the <a>accessibility tree</a> for <a>Text node</a>s.</p>
```

Comment:
Is this necessary? Seems like it could be omitted.

Reply:
@cookiecrook 
The history of that line is in https://github.com/w3c/aria/pull/950#discussion_r286189188.
Summary:
- The [spec currently says that text nodes MUST be in the a11y tree](https://w3c.github.io/aria/#tree_inclusion).
- @joanmarie said that on some platforms, the text is moved up into the parent node, and the text node itself is not included in the tree, so need to remove the MUST.
- I changed it to MAY.

I'm not sure whether the line should be omitted, given that it used to say MUST... perhaps we should leave it in with MAY so that implementers know what happened to it? @joanmarie, do you have any comment about this?

Reply:
As discussed during the ARIA call, I'm good with removing it.